### ISSUE TITLE
Write ISO to USB drive

### ISSUE THREAD

**Issue by jagauthier** [rear/rear#2581]
#### Relax-and-Recover (ReaR) Issue Template

Fill in the following items before submitting a new issue
(quick response is not guaranteed with free support):

* ReaR version ("/usr/sbin/rear -V"):
2.4 

* OS version ("cat /etc/os-release" or "lsb_release -a" or "cat /etc/rear/os.conf"):
CentOS 7.8

This is a simple inquiry.
I am writing the recovery ISO to disk.  I can write this recovery ISO to a USB stick using Rufus under Windows.  IT boots fine and recovery is possible.

However, I noticed the ISO is just a partition.  I am trying to figure out how to write the ISO from a linux system to USB drive.
I am aware that I can do OUTPUT=USB, but I don't want to leave a UB drive in all the time.

Basically, it needs to be a EUFI bootable drive and I can't seem to figure this out from the linux command line.  The way I've written raw disk images in the past (dd) doesn't work.  Mostly because it appears the ISO is just a partition.

Thanks for any ideas.

**Comment by jagauthier** [comment]
Well, I'll leave this here.

From a USB drive as FAT32.  Extract the ISO to the drive. EUFI looks for /boob/efi/bootx64.efi

That's it.

**Comment by jsmeix** [comment]
@jagauthier 
perhaps https://github.com/rear/rear/issues/2210
could be also of interest for you here?

Additionally there is the `RAWDISK` output method, cf.
https://github.com/rear/rear/pull/2555#issuecomment-767451545
but `RAWDISK` does not support to include the data backup, cf.
https://github.com/rear/rear/pull/2555#issuecomment-769034845
and
https://github.com/rear/rear/pull/2555#issuecomment-769144572

**Comment by jagauthier** [comment]
Thank you! I didn't know about RAWDISK. I will try it for sure.

**Comment by jsmeix** [comment]
@jagauthier 
there are some recent improvements of the `RWADISK` method
all done by @OliverO2 in our current ReaR upstream master code
so I recommend to try out our current ReaR upstream master code,
cf. "Testing current ReaR upstream GitHub master code" in
https://en.opensuse.org/SDB:Disaster_Recovery

**Comment by speefak** [comment]
simple and fast solution to create an bootable USB Stick from a non hybrid ISO file : https://askubuntu.com/a/1022014
